welcome-to-sourcery.py

```diff
@@ -1,13 +1,76 @@
 
-# Welcome to Sourcery!
-
-# To get started log into your Sourcery account.
-# Open the command palette (Ctrl/Cmd+Shift+P) and execute `Sourcery: login`.
-
-# We're here to help you write cleaner, clearer code.
-# When you log in to Sourcery you'll start to see parts 
-# of your code like this start to get underlined. 
-# This means Sourcery has a suggestion.
+# Welcome to Sourcery! We're here to be your pair programmer
+# anytime you're working in VS Code.
+
+# To get started log into your Sourcery account. Click on
+# the Sourcery logo (the hexagon) on your VS Code sidebar
+# and click the button to log in.
+
+# Or, open the command palette (Ctrl/Cmd+Shift+P) and
+# execute `Sourcery: login`.
+
+# Sourcery works in 2 ways:
+# 1. Gives you instant suggestions for improvements and
+# refactorings to your Python, JavaScript, and TypeScript
+# code. All of this runs fully locally
+# 2. Acts as a pair programmer allowing you to ask it
+# questions, write new code, and interact with existing code.
```

Comment:
I would put `using AI` or something in here and make it a bit more explicit that this doesn't run locally